packages/core/src/shared/clients/docdbClient.ts

```diff
@@ -13,7 +13,7 @@ import * as DocDBElastic from '@aws-sdk/client-docdb-elastic'
 
 const DocDBEngine = 'docdb'
 
-function isElasticCluster(clusterId: string): boolean {
+function isElasticCluster(clusterId: string | undefined): boolean | undefined {
```

Comment:
You seem to be using this in ways that evaluate its truthiness. The result of `undefined` is always interpreted as `false`. Should you just return `false` in this case and remove the possibility of `undefined` result?

Reply:
I thought about that.  But I wanted to return undefined if the input is undefined (which should never happen in theory)